user-stories/US-07-Privacy-request-capture-service.md

```diff
@@ -0,0 +1,316 @@
+
+### US-07-1-07
+
+**As an** Internet User (a Submitter or a Data Subject),
+
+**I want to** submit a Privacy Request to modify my address and to modify a field from an allergy record
+
+**because** because I just moved and also discovered I have a new allergy
+
+### US-07-1-08
+
+**As an** Internet User (a Submitter or a Data Subject),
+
+**I want to** submit a Privacy Request to access some specific data : the data that are derived from me and my behavior  
+
+**because** I want to know what data a system has one me that I did not input myself
+
+### US-07-1-09
+
+**As an** Internet User (a Submitter or a Data Subject),
+
+**I want to** submint a Privacy Request to know the [provenance](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/dictionary/provenance/en.provenance.json) of the data a system has on me
+
+**because** I suspect someone might have shared my data without my authorization
+
+### US-07-1-10
+
+**As an** Internet User (a Submitter or a Data Subject), submitting a Privacy Request to an Organization
+
+**I want** my request to be taken into account by some or all possible targets from this [targets list](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/dictionary/targets/en.targets.json)
+
+**because** I want my Privacy Request to be fully completed
+
+### US-07-1-11
+
+**As an** Internet User (a Submitter or a Data Subject), when creating a Privacy Request
+
+**I want to** be guided in making my Privacy Request as precise and as correct as possible
+
+**because** I want or avoid making a Privacy Request that is likely to be rejected (imprecise, or wrongly formulated)
+
+### US-07-1-12
+
+**As an** Internet User (a Submitter or a Data Subject), when creating a Privacy Request
+
+**I want to** be guided towards making a Privacy Request in a way that makes it automatically processable
+
+**because** I want or avoid making a Privacy Request that takes too much time (and human intervention) to process
+
+### US-07-1-13
+
+**As an** Internet User (a Submitter or a Data Subject), when creating a Privacy Request
+
+**I want to** remain anonymous or at least not prove my identity
+
+**because** My Privacy Request only concerns TRANSPARENCY about data usage (under Art. 13 and 14. of GDPR)
+
+### US-07-1-14
+
+*(see [a priori](../refs/schemas/priv/scenarios.md#a-priori-authentication) and [a posteriori](../refs/schemas/priv/scenarios.md#a-posteriori-authentication) authentication)*
+
+**As an** Internet User (a Submitter or a Data Subject), when creating a Privacy Request
+
+**I want to** authenticate
+
+**because** I am requesting something that requires proof of identity
+
+### US-07-1-15
+
+**As an** Internet User (a Submitter or a Data Subject), after creating a Privacy Request
+
+**I want to** know the status of my Privacy Request
+
+**because** I have been waiting for a while and have got no response yet
+
```

Comment:
```suggestion
### US-07-1-16

**As an** Internet User (a Submitter or a Data Subject), after creating a Privacy Request

**I want to** know the see/download the data concerning me

**because** the Privacy Request I made included an `ACCESS` demand.
```